### PR TITLE
#79: Removed Media module

### PR DESCRIPTION
**Pull request by ulin-evgeny**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
     "nunomaduro/collision": "^6.4",
     "php-open-source-saver/jwt-auth": "^2.2",
     "psr/simple-cache": "^3.0",
-    "ronasit/laravel-helpers": "dev-master",
+    "ronasit/laravel-helpers": "^2.5",
+    "ronasit/laravel-media": "^0.0.1@beta",
```
Comment (DenTray) [inline composer.json]: we no need to media module installed by default, let's remove it

**Review by DenTray — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (ulin-evgeny) [reply composer.json]: Done. Completely removed Media from the project.

File: config/app.php [hunk 2]
```diff
@@ -207,7 +207,6 @@
         /*
          * Module Service Providers...
          */
```
Comment (DenTray) [inline config/app.php]:
```suggestion
```

**Review by DenTray — DISMISSED** [review]

File: config/app.php [hunk 3]
```diff
@@ -204,10 +204,6 @@
         RonasIT\Support\AutoDoc\AutoDocServiceProvider::class,
         PHPOpenSourceSaver\JWTAuth\Providers\LaravelServiceProvider::class,
```
Comment (DenTray) [inline config/app.php:206]:
```suggestion
```

**Review by DenTray — DISMISSED** [review]

**Review by DenTray — APPROVED** [review]

**Pull request merged**